### DISCUSSION TITLE
using-wireguard/

### DISCUSSION THREAD

**Discussion by giscus** [fr0der1c/dev-blog-discussions#4] [Announcements]
# using-wireguard/

不知道从什么时候开始，B 站博主和什么值得买的值友都开始纷纷带货 NAS 了。看了很多人推荐，于是自己也买了一台群晖 NAS 来玩玩。由于 NAS 部署在家中，出门的时候要访问文件就非常不方便。传统的做法是使用群晖的 QuickConnect 或者 DDNS。然而，前者（据说）速度很慢，后者将设备暴露在公网上，安全性又存在一些疑虑。因此我决定试一试组网工具 WireGuard。

https://dev.admirable.pro/using-wireguard/

**Comment by DaXiongAhey** [comment]
你好，想问一下，公私钥都在服务端上生成吗，客户端不需要生成吗？
